experimental/packages/sampler-jaeger-remote/package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "@opentelemetry/sampler-jaeger-remote",
+  "version": "0.52.1",
+  "description": "Jaeger Remote Sampler",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "prepublishOnly": "npm run compile",
+    "compile": "tsc --build",
+    "clean": "tsc --build --clean",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc mocha 'test/**/*.test.ts' --exclude 'test/browser/**/*.ts'",
+    "version": "node ../../../scripts/version-update.js",
+    "watch": "tsc -w",
+    "precompile": "cross-var lerna run version --scope $npm_package_name --include-dependencies",
+    "prewatch": "npm run precompile",
+    "peer-api-check": "node ../../../scripts/peer-api-check.js",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../../"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nodejs",
+    "metrics",
+    "stats"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@opentelemetry/sdk-trace-base": "1.25.1"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": ">=1.3.0"
```

Ah, this is semantically different to `^1.3.0`. In its current state it will also accept `2.0.0`, `3.0.0` which may break. So I'd suggest:
```suggestion
    "@opentelemetry/api": "^1.3.0"
```